azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -39,15 +37,15 @@
 _logger = getLogger(__name__)
 
 
-_SUPPORTED_INSTRUMENTED_LIBRARIES = (
-    "django",
-    "fastapi",
-    "flask",
-    "psycopg2",
-    "requests",
-    "urllib",
-    "urllib3",
-)
+_SUPPORTED_INSTRUMENTED_LIBRARIES_TO_INSTRUMENTS_MAP = {
```

Comment:
This languages follows how the instrumentors' fields are set ups, but it may be clearer to user ..._to_optional_dependencies or something like that.

Reply:
I'm not sure I understand this statement.

Reply:
Changing to _SUPPORTED_INSTRUMENTED_LIBRARIES_DEPENDENCIES_MAP for clarity